src/components/AnalyticsDrawer/AnalyticsDrawer.tsx

```diff
@@ -1,30 +1,52 @@
+import CircularProgress from 'material-ui/CircularProgress';
 import { Drawer } from 'material-ui'
 import MarkerIcon from 'mui-icons/cmdi/map-marker'
 import * as React from 'react'
 import { Link } from 'react-router-dom'
-import { withStore } from '../../services/store'
+import { store, withStore } from '../../services/store'
 import { InfoBox } from '../InfoBox/InfoBox'
 import { ServiceAreaSelector } from '../ServiceAreaSelector/ServiceAreaSelector'
+import { Refresh } from '../Refresh/Refresh'
 import './AnalyticsDrawer.css'
 import { ServiceAreaAnalytics } from './ServiceAreaAnalytics'
 import { TotalAnalytics } from './TotalAnalytics'
 
+function pickAnalytics() {
+  if (store.get('serviceAreas').length == 0) {
+    return <InfoBox large>
+        Please choose a service area in the <Link to='/service-areas'><MarkerIcon /> Service Areas drawer</Link>
+      </InfoBox>
+  }
+  if (store.get('adequacyComputationInProgress')) {
+    return <div style={{marginTop: 30}}>
+      <div style={{marginLeft: 'auto', marginRight: 'auto', display: 'block', width: 140}}>
+        <div style={{marginBottom: 15, marginTop: 15}}>Computing adequacy...</div>
+        <CircularProgress size={120} thickness={7} style={{display: 'block'}} />
+      </div>
+    </div>
```

Comment:
Please move all of these inline styles to CSS.. Inline styles are:

- Impossible to debug
- Difficult to reuse
- Difficult to compose
- Difficult for the rendering engine to optimize
- Difficult for designers to review